### ISSUE TITLE
Add Different Time Customizations 

### ISSUE THREAD

**Issue by Fogest** [FreshRSS/FreshRSS#3103]
Currently other platforms like tt-rss and other non-self hosted RSS readers will have times in a relative format. Currently I have to look at the time and take a rough guess in my head on how old an article is. Whereas if I look in something like tt-rss I can see that an article was imported "23 minutes ago".

After it is over 1-2 hours old it switches to just showing a time such as "7:03" and "10:21". If the feed was imported from a previous day other than today the date/time would switch to show "Jul 11, 23:49". 

Doing the time in a way similar to tt-rss or other readers allows you to have to do less manual comparing to what your current time is as the website can do it for you. 

I would think a setting like this may be suited for the "Display" configuration.

**Comment by aledeg** [comment]
I think this is probably better as an extension.

**Comment by aledeg** [comment]
@Fogest I've just added an extension to customize how dates are displayed. It's available [here](https://github.com/aledeg/FreshRSS-extensions/tree/master/xExtension-DateFormat). Let me know what you think.

**Comment by Fogest** [comment]
@aledeg Awesome, it didn't work at first but I noticed a bunch of commits that reference this which I assume were needed for the extension. I updated to the `:latest` on docker and the extension works great!

I am going to close this, however as a secondary question, is there a reason when supporting by unread that there is not an order to the unread (see image below)? My `sort order` in the settings is using "Newest first"

![image](https://user-images.githubusercontent.com/1163628/87627510-e4bef700-c6fc-11ea-8905-2658c45bec76.png)

(also not sure why that one is happening in the future)
